### DISCUSSION TITLE
Can crio.service be executed as non-root user?

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#8012] [Q&A]
Can crio.service be executed as non-root user? If yes, how to do that?
Also, how will it impact containers and any other service or config?

**Comment by alpana17** [comment]
@saschagrunert Please suggest

**Comment by saschagrunert** [comment]
Hey, no, rootless CRI-O is right now not supported.

**Reply by alpana17** [reply]
ok
Thank you
